test/hash_test.rb

```diff
@@ -3,6 +3,8 @@
 require 'hanami/utils/hash'
 
 describe Hanami::Utils::Hash do
+  CURRENT_RUBY_VERSION = defined?(TRAVIS_RUBY_VERSION) ? TRAVIS_RUBY_VERSION : RUBY_VERSION
```

Comment:
@davydovanton Is this new constant defined because Travis CI doesn't support 2.4 yet? Can we get rid of this?